.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,48 @@
+name: "Triage Stale issues"
+
+on:
+  schedule:
+  - cron: "0 * * * *"
+
+jobs:
+  issue-cleanup:
+    runs-on: ubuntu-latest
+    name: Triage Stale Issues
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v4
+      with:
+        issue-types: issues
+        ancient-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer 
+          than five months. We encourage you to check if this is still an issue in the latest release. 
+          In the absence of more information, we will be closing this issue soon. 
+          If you find that this is still a problem, please feel free to provide a comment or upvote 
+          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
+          please feel free to open a new one.
+        stale-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer than two months. 
+          We encourage you to check if this is still an issue in the latest release. 
+          In the absence of more information, we will be closing this issue soon. 
+          If you find that this is still a problem, please feel free to provide a comment or upvote 
+          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
+          please feel free to open a new one.
+        # These labels are required
+        stale-issue-label: closing-soon
+        exempt-issue-labels: bug, needs-triaging, should-be-fixed
+        response-requested-label: response-requested
```

Comment:
Maybe also add `awaiting-response` here? (which is a label that has already been attached to a couple of issues). Or do we deliberately want to create a new label here, to start with a fresh status for the workflow of all tickets?

Reply:
Ah, so this was my misunderstanding - thought the purpose was to map this also existing labels (but as the singular name `response-requested-label` implies this should be a single one). Makes sense to rename `response-requested` with `awaiting-response` in the config here (or alternatively leave the PR as-is and switch `awaiting-response`->`response-requested` in the existing issue backlog). 👍